### PR TITLE
Fix Background and fit property fixes #268

### LINKED ISSUE #268: Background doesn't support changing fit
**Describe the bug**
Setting a imgixParams fit to anything but crop has no effect. It still is set to crop as the fit is hardcoded. It should be able to be override by passing a imgixParams fit value.

**To Reproduce**
https://codesandbox.io/s/540nzkv4qn

Steps to reproduce the behaviour:

1.  Create a new Background element
2.  Set imgixParams.fit to anything but crop
3.  See URL has fit=crop

**Expected behaviour**
Fit should change depending what is set in imgixParams

**Information:**

- react-imgix version: latest

### PR DESCRIPTION
**Pull request by verbeeksteven**
Fixes #268

### CODE REVIEW

**Comment by verbeeksteven** [comment]
@sherwinski Yup! I wasn't sure if there was something else you wanted done. You are more then welcome to adjust/do what ever you want with my PR :)

**Comment by verbeeksteven** [comment]
Any update @sherwinski I can make adjustments if you want.

File: src/react-imgix-bg.jsx [hunk 1]
```diff
@@ -88,7 +88,7 @@ const BackgroundImpl = props => {
       ...(disableLibraryParam ? {} : { ixlib: `react-${PACKAGE_VERSION}` }),
       width,
       height,
-      fit: "crop",
+      fit: (imgixParams.fit) ? imgixParams.fit : "crop",
```
Comment (sherwinski) [inline src/react-imgix-bg.jsx]: If I could suggest tweaking things here for cleanliness:
```
const renderedSrc = (() => {
   const srcOptions = {
     fit: "crop",
     ...imgixParams,
     ...(disableLibraryParam ? {} : { ixlib: `react-${PACKAGE_VERSION}` }),
     width,
     height,
     dpr
   };

   return constructUrl(src, srcOptions);
 })();
```

**Comment by sherwinski** [comment]
Hey @verbeeksteven sorry for the silence on this PR but don't worry, I did not forget about you! I am going to fix this issue as part of a separate PR, mostly to sidestep issues with prettier but also to add tests and whatnot. I'd love to add you as a reviewer if you'd like to take a look at the end result. Feel free to comment back with any questions.

**Comment by verbeeksteven** [comment]
I haven't had a ton of time to do it either, I will certain test out any edits you do though!

**Pull request closed without merging**